pandunia-loge.csv

```diff
@@ -1171,6 +1183,7 @@ hasem|mate|Hs||hassium||hasio|||||||||||||hasio|hasium|has
 hashish||||hashish||hachís||||||||||||||hasis|haszysz
 hasi|||ara:(xāṣṣ), fas:urd:(xās), hin:ख़ास (xās), may:khas|special (distinct)||especial (diferenciado, distinto)|||||||||||||speciala|erikoinen (erityinen)|specjalny, wyraźny, wyrazisty
 hasia||||distinction (specialty)||especialización (distinción)||||||||||||||erikoisuus|różnica (specjalność)
+hasnam||||identifier (ID)||identificador||идентификатор|||識別名||||||||||
```

Comment:
haskode?

Reply:
"haskode" multo anklari, sa mi ray. "haskode" abla esa "cipher" o "encryption key" o "personal coupon". the key feature of an identifier is that it allows one to reference something complicated with a small amount of information, like a name does for people and places.